src/main/java/de/metas/ui/web/handlingunits/process/WEBUI_M_HU_MoveToAnotherWarehouse.java

```diff
@@ -46,76 +28,22 @@
  * #L%
  */
 
-/**
- * #2144
- * HU editor: Move selected HUs to another warehouse
- *
- * @author metas-dev <dev@metasfresh.com>
- *
- */
-public class WEBUI_M_HU_MoveToAnotherWarehouse extends HUEditorProcessTemplate implements IProcessPrecondition
+public class WEBUI_M_HU_MoveToAnotherWarehouse extends WEBUI_M_HU_MoveToAnotherWarehouse_Helper
 {
-	private final transient IHUMovementBL huMovementBL = Services.get(IHUMovementBL.class);
-	private final transient IHandlingUnitsDAO handlingUnitsDAO = Services.get(IHandlingUnitsDAO.class);
-	private final transient IHUStatusBL huStatusBL = Services.get(IHUStatusBL.class);
-
-	@Param(parameterName = I_M_Warehouse.COLUMNNAME_M_Warehouse_ID, mandatory = true)
-	private I_M_Warehouse warehouse;
-
-	private HUMovementResult movementResult = null;
+	private final transient HULotNumberLockService lotNumberLockService = Adempiere.getBean(HULotNumberLockService.class);
 
 	@Override
 	protected ProcessPreconditionsResolution checkPreconditionsApplicable()
```

Comment:
please somehow make sure that the `doIt()` method repeats the "quarantine" check of the "checkPreconditionsApplicable()" method.
Because the HU might have been changed in between.
Maybe add an abstract method like "assertHUsEligible()" to `WEBUI_M_HU_MoveToAnotherWarehouse_Helper`

Reply:
hi @metas-rc it seems as if you overlooked this one. can you please chack?